common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -30,6 +31,7 @@
 import net.minecraft.client.renderer.MultiBufferSource;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
+
```

Comment:
Spotless ought to have caught this?

Reply:
![image](https://github.com/Wynntils/Artemis/assets/51191570/03215b21-c3b7-40c7-9ba5-fdc668df9bcb)
It didn't.
Should it be 1 or 2 spaces?

Reply:
One line. And it complains to me in my clone if I add an additional newline here.. weird.